python/src/ParametricFunction_doc.i.in

```diff
@@ -23,6 +20,11 @@ parametersSet : bool, optional
     Otherwise, the set variables are the ones referenced in the complementary
     set of *indices*.
 
+Notes
+-----
+A ParametricFunction defines a parametric function from *function* by freezing the variables
+marked by the *indices* set to the values of *referencePoint*.
+
```

Comment:
maybe "partial function" is a better term than parametric here what do you think ? its also the math term

Reply:
ok! Good idea!